special-pages/pages/history/app/mocks/mock-transport.js

```diff
@@ -70,30 +104,44 @@ export function mockTransport() {
                     }
                     return Promise.resolve({ action: 'none' });
                 }
-                case 'title_menu': {
-                    console.log('📤 [deleteRange]: ', JSON.stringify(msg.params));
+                case 'deleteRange': {
+                    // console.log('📤 [deleteRange]: ', JSON.stringify(msg.params));
                     // prettier-ignore
                     const lines = [
-                        `title_menu: ${JSON.stringify(msg.params)}`,
+                        `deleteRange: ${JSON.stringify(msg.params)}`,
                         `To simulate deleting this item, press confirm`
-                    ].join('\n');
+                    ].join('\n',);
                     if (confirm(lines)) {
+                        if (msg.params.range === 'all') memory = [];
                         return Promise.resolve({ action: 'delete' });
                     }
                     return Promise.resolve({ action: 'none' });
                 }
-                case 'deleteRange': {
-                    console.log('📤 [deleteRange]: ', JSON.stringify(msg.params));
+                case 'deleteDomain': {
+                    // console.log('📤 [deleteDomain]: ', JSON.stringify(msg.params));
```

Comment:
I'm assuming that commenting out some `console.log('📤 [...]` but not others is intentional